### ISSUE TITLE
CopyButton accessibility violation on visible label

### ISSUE THREAD

**Issue by nictownsend** [carbon-design-system/carbon#6009]
> Automated testing tool and ruleset
DAP - IBM Accessibility January 2020

## Detailed description

Copy component uses 'feedback' property as the assistive span content - which is hidden via css, but is flagged as a visible label (and not matching the aria-label).

Might be a false positive in DAP.

> What version of the Carbon Design System are you using?
10.11.2

> What WCAG 2.1 checkpoint does the issue violate?
2.5.3 https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html

https://codesandbox.io/s/magical-feynman-6x7sm

## Additional information

![image](https://user-images.githubusercontent.com/47386186/80992495-8094d280-8e31-11ea-8b2b-38b0597d377c.png)

**Comment by emyarod** [comment]
after testing with VO it seemed the correct labels were being read and the assistive span content is read only when the tooltip is visible. does that make this a false positive or is there some additional action we can take here?

**Comment by nictownsend** [comment]
So I think the error is flagged so that if you navigate by voice, you have to be able to say the visible label. This doesn't have one - but I think the tool is flagging it because the assistive span is hidden via CSS, but the DAP isn't spotting it as such

**Comment by jrpomeroy** [comment]
The problem is that the span with the assistive text (Copied!) is part of the content of the button, so it's being treated by DAP as the "visible label" even when it's not technically visible. This text probably should not be part of the button content, but placed after the button.

**Comment by jhpedemonte** [comment]
We ran into this when doing a scan using the Accessibility Checker extension. Agreed with @jrpomeroy's proposed solution.

**Comment by jay-mack** [comment]
Having the same problem.

<img width="608" alt="Screenshot 2020-06-12 at 14 39 33" src="https://user-images.githubusercontent.com/66827828/84508685-94e2af80-acba-11ea-99c9-6bb9f44eb845.png">

I think this is kind of similar to another problem I encountered however the svg had an aria-label so was able to set the button aria label equal and passed the accessibility scan.

<img width="517" alt="Screenshot 2020-06-12 at 12 13 48" src="https://user-images.githubusercontent.com/66827828/84508367-14bc4a00-acba-11ea-9c86-143b26c51194.png">
